assets/css/style.css

```diff
@@ -0,0 +1,152 @@
+html {
+  font-family: "Poppins", serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  min-width: 375px;
+  max-width: 100%;
+  background-image: url('../images/header-background-mobile.svg');
+  background-repeat: no-repeat;
+  background-size: contain;
+  overflow-x: hidden;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  margin-top: 10px;
+  padding: 10px;
+}
+
+.header-logo {
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  line-height: 20px;
+  padding: 3px 5px;
+}
+
+.menu-icon {
+  height: auto;
+  padding: 3px 5px;
+}
+
+.menu-icon:hover {
+  cursor: pointer;
+}
+
+h1 {
+  color: #1193f6;
+}
+
+header > nav > a:hover {
+  text-decoration: underline;
+}
+
+.menubar {
+  background-color: transparent;
+  margin-right: 0.5em;
+}
+
+main {
+  min-height: 85vh;
+}
+
+.content {
+  min-height: 80vh;
+  z-index: 1;
+  position: absolute;
+  padding: 0 1em;
+}
+
+.content > h1 {
+  margin-top: 2.5em;
+  margin-bottom: 0;
+  width: 327px;
+  max-width: 100%;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.content > div > p {
+  margin-top: 0;
+  margin-bottom: 1em;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.content > h4 {
+  padding-top: 0;
+  margin-bottom: 0;
+  height: 24px;
+  font-family: 'Poppins', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.content > ul {
+  list-style: none;
+  display: flex;
+  padding: 0;
+  margin-top: 0.3em;
+}
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 add the space bellow the social icons to be similar to the Figma design 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://github.com/chudisoft/Portfolio-Microverse/assets/58642949/8be8cacd-06fb-49fa-9676-6b5f4d16f97b) | ![image](https://github.com/chudisoft/Portfolio-Microverse/assets/58642949/7d6da23c-6714-4796-91f6-13efe61f62d3) |